awx/ui_next/src/screens/User/UserList/UserList.test.jsx

```diff
@@ -255,17 +246,7 @@ describe('UsersList with full permissions', () => {
   });
 
   test('Add button shown for users with ability to POST', async () => {
-    await waitForElement(
-      wrapper,
-      'UsersList',
-      el => el.state('hasContentLoading') === true
-    );
-    await waitForElement(
-      wrapper,
-      'UsersList',
-      el => el.state('hasContentLoading') === false
-    );
-    expect(wrapper.find('ToolbarAddButton').length).toBe(1);
+    await waitForElement(wrapper, 'ToolbarAddButton', el => el.length === 1);
```

Comment:
Checking that the final, expected state eventually happens (ToolBarAddButton is visible) is enough for this test.